kura/org.eclipse.kura.web2/pom.xml

```diff
@@ -252,7 +252,7 @@
 				<artifactId>maven-compiler-plugin</artifactId>
 				<version>3.3</version>
 				<configuration>
-					<source>1.6</source>
+					<source>1.7</source>
```

Comment:
It is a leftover from a test. But binary code already is 7. So that setting doesn't change anything.